cli/cli.go

```diff
@@ -164,6 +164,10 @@ func extractBinary(source *bytes.Reader, os string) (binary io.ReadCloser, err e
 		}
 
 		for _, f := range zr.File {
+			info := f.FileInfo()
+			if info.IsDir() || !strings.HasSuffix(f.Name, ".exe") {
```

Comment:
Why iterating over all files?

Why not just extract `exercism.exe`?

Reply:
> Why iterating over all files?

I don't want to assume that the first file is the executable, even though that is probably the case.

> Why not just extract exercism.exe?

If I extract the whole zip I would have to create the structure locally, then write each of the files. I am essentially extracting the executable without writing the file to disk (less to clean up). I could check for the exact name, read it's contents, then return in case we decide to add more binaries to the release package.

Reply:
Anytime! Thanks for the question and review.